graphql/typeDefs.ts

```diff
@@ -112,11 +112,11 @@ export default gql`
 
   type User {
     id: Int!
-    username: String
+    username: String!
     userLesson: UserLesson
-    email: String
-    name: String
-    isAdmin: String
+    email: String!
+    name: String!
+    isAdmin: Boolean!
```

Comment:
Until we make the `username`, `email` and `name` fields non-nullable on the database, this will conflict with the return types from prisma queries, and putting `as any` after it is not a good solution.